### ISSUE TITLE
Fix decay

### ISSUE THREAD

**Issue by qxcv** [qxcv/joint-regressor#22]
Right now, reported learning rate is staying constant over time. That indicates that learning rate is either staying constant (bad) or not being calculated incorrectly (not as bad, but still annoying). See [this issue](https://github.com/fchollet/keras/issues/888) and [this issue](https://github.com/fchollet/keras/issues/109) for relevant commentary.

**Comment by qxcv** [comment]
Should be fixed by 3bdba288ca443ed42aa7d40b533ac64105af26d7
